dnn/variational_inference.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        df = df[df[name].notnull()]
</pre><p>do we want to print out rows dropped here?  how would we catch a regression when a geotiff returns incorrect nulls?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
At the end of the cell, we print out the list of columns fed into training.